### ISSUE TITLE
Cannot install Starcounter on Parallels

### ISSUE THREAD

**Issue by k-rus** [Starcounter/Starcounter#3570]
It seems that the installer is broken. By default the installer wants to install Starcounter into the user folder, which is usually a mounted network folder on Parallels (*\\psf\Home\Documents\Starcounter*). I manually change it to a folder on local drive *C*. However, the installer ignores my choice and creates the server folder in the default folder, i.e., the network one. And Starcounter do not work with such folders at all.

This bug was introduced recently. I guess it was during fixing another issue without doing proper testing.

**Comment by k-rus** [comment]
Tested with *Starcounter-DevelopNightlyBuilds-2.2.1.2038-Setup*

**Comment by dan31** [comment]
This might be related to https://github.com/Starcounter/Starcounter/issues/3535.

**Comment by k-rus** [comment]
> This might be related to #3535.

Exactly. I guess the fix of #3535 was not properly tested and introduced this bug.

**Comment by andwah** [comment]
I did some investigating and it turns out it's been a bug since day one, so it's not related to #3535.

When an warning was shown in the installer (a yellow circle beside the input field) the entered value was ignored and previous value was then used.

**Comment by dan31** [comment]
This must be a duplicate of https://github.com/Starcounter/Starcounter/issues/3560.

**Comment by andwah** [comment]
I pushed a fix 38f211cdd8f4b98a938575b7d5698174bcb92e7e into develop. Needed to redo on how warnings was treated.
